config.minimal.yml

```diff
@@ -5,7 +5,7 @@
 authentication_backend:
   file:
     # The path to the database file. The file is at the root of the repo.
-    path: /etc/authelia/users_database.yml
+    path: /etc/authelia/volume/users_database.yml
```

Comment:
I think you can avoid that.

Reply:
This configuration will be reused in a demo mode of Authelia (to be implemented) where users won't even need to have a nginx or docker installed on the machine. In that case 'volume' makes no sense.

Reply:
I did it like this because in docker the volume that is mounted should preferably be in a subdirectory of /etc/authelia, because that way you only need one single docker volume instead of two. (for users_database.yml and storage each)

Its of course possible to do it with two volumes and without a subdirectory.